pkg/operator/configmap/configmap_controller.go

```diff
@@ -108,6 +109,7 @@ type ReconcileConfigMap struct {
 	client.Client
 	logger            log.FieldLogger
 	configMapDataHash string
+	context           context.Context
```

Comment:
@suhanime it is advised not to store a context inside a struct. Any specific reason why we are doing it here?

https://github.com/golang/go/blob/2887ef499a0e01a3b6fe92bf874843f9f2d05167/src/context/context.go#L29-L36

Reply:
For the case (not implemented in this PR) where we want to pass the context generated up in https://github.com/openshift/cloud-credential-operator/blob/master/pkg/cmd/operator/cmd.go down into a runtime controller, where would we otherwise store it?